### DISCUSSION TITLE
GraphQL query for fetching user's emails with user:email scope

### DISCUSSION THREAD

**Discussion by nikochiko** [community/community#24389] [Apps, API and Webhooks]
<p>Hello!<br>
I want to get the user’s emails (<code>GET /user/emails</code> on REST) from the GraphQL API.<br>
I did not find any documentation about this, how can I do it?</p>
<p>Note: I want to get all of the user’s emails connected on GitHub and not just the publicly visible one.</p>

**Comment by byrneh** [comment, marked as answer]
<p>Unfortunately every REST v3 API does not necessarily have an equivalent GraphQL feature.</p>
<p>In this example you are stuck with <a href="https://docs.github.com/en/rest/reference/users#emails" rel="nofollow noopener">users#emails</a> from REST v3 API as the only way of retrieving this information unless in future it is added GraphQL.</p>

**Reply by kzu** [reply]
Please consider adding support for this 🙏
